Source/WebGPU/WebGPU/Buffer.h

```diff
@@ -123,7 +123,9 @@ class Buffer : public WGPUBufferImpl, public RefCounted<Buffer>, public CanMakeW
     void incrementBufferMapCount();
     void decrementBufferMapCount();
 
+PUBLIC_IN_WEBGPU_SWIFT
```

Comment:
This is fine, but I think we can also have an idiom going forward where we always write PUBLIC_IN_WEBGPU_SWIFT right next to the existing qualifier. 

The net effect will make more things public in WebGPU Swift, but it will be cleaner, and once we commit to exposing private stuff, I don’t see any value in being selective about it.